app/models/questionnaire_version.rb

```diff
@@ -0,0 +1,29 @@
+class QuestionnaireVersion < ActiveRecord::Base
+  has_one :questionnaire, required: false
+
+  def localized_layout
```

Comment:
What do you think about moving the localisation logic to the transformers?

Reply:
Localization is a model-level concern. Keeping it in model enables reuse in future on dashboards, reports, etc.